xds/src/main/java/io/grpc/xds/Bootstrapper.java

```diff
@@ -53,17 +53,22 @@
 
   private static final Bootstrapper DEFAULT_INSTANCE = new Bootstrapper() {
     @Override
-    public BootstrapInfo readBootstrap() throws IOException {
+    public BootstrapInfo readBootstrap() throws XdsInitializationException {
```

Comment:
As mentioned elsewhere, is it possible to cache `BootstrapInfo` value so this method doesn't reread the file every single time? Or are there use-cases to support this flow?